cooluri/link.Functions.php

```diff
@@ -259,7 +259,7 @@ public static function pageNotFound($lConf) {
       if (!empty($lConf->cache->pagenotfound)) {
           switch ((string)$lConf->cache->pagenotfound->behavior['type']) {
               case 'message': $res = (string)$lConf->cache->pagenotfound->behavior; break;
-              case 'page':    $res = file_get_contents((string)$lConf->cache->pagenotfound->behavior); break;
+              case 'page':    $res = t3lib_div::getUrl((string)$lConf->cache->pagenotfound->behavior); break;
```

Comment:
Hmm... than you have to write wrapper classes... there is also a mysql_error call in this class that is useless in newer PHP and TYPO3 Versions... main problem is, that many hosters do not allow to fetch external pages via file_get_contents. That is the reason that curl take care of this problem. Are there any plans to build a wrapper for this too?